openbas-api/src/main/java/io/openbas/rest/exercise/ExerciseApi.java

```diff
@@ -220,6 +220,7 @@ public List<ComcheckStatus> comcheckStatuses(
   // endregion
 
   // region teams
+  @LogExecutionTime
   @GetMapping(EXERCISE_URI + "/{exerciseId}/teams")
   @PreAuthorize("isExerciseObserver(#exerciseId)")
```

Comment:
**suggestion (non-blocking) :** 
```suggestion
  @PreAuthorize("isExerciseObserver(#exerciseId)")
  @Tracing(name = "Get teams of the exercise", layer = "api", operation = "GET")
```
Adding those will allow us to have a better view of how long the API takes on each layer during a call. That can help in the long run (this is totally optional for this PR)